src/kbmod/analysis/visualizer.py

```diff
@@ -0,0 +1,113 @@
+from kbmod.analysis.plotting import plot_multiple_images
+from kbmod.search import StampCreator
+from .visualization_utils import *
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+
+class Visualizer:
+    """A class for visualizing from a given `ImageStack` and `Results` set.
+
+    Attributes
+    ----------
+    im_stack : `kbmod.search.ImageStack`
+        `ImageStack` associated with the results.
+    results : `kbmod.Results`
+        The loaded `Results`.
+    """
+
+    def __init__(self, im_stack, results):
+        self.im_stack = im_stack
+        self.results = results
+        self.trajectories = results.make_trajectory_list()
+
+    def generate_all_stamps(self, radius=10):
+        """Creates a stamp cutout for each image for each
+        result. Also creates a new column in the results
+        table called `all_stamps`.
+
+        Parameters
+        ----------
+        radius : `int`
+            radius of the stamp.
+        """
+        self.results.table["all_stamps"] = [
+            StampCreator.get_stamps(self.im_stack, trj, radius) for trj in self.trajectories
+        ]
+
+    def count_num_days(self):
+        """Counts the number of days that a given result
+        has valid data in.
```

Comment:
Can you add a sentence that it saves this internally to a new column in the results?